pkg/cmd/application/create/app_create.go

```diff
@@ -60,10 +61,16 @@ NOTE:
 			} else {
 				opts.LocalPath = "./" + opts.Name
 			}
+
+			nonInteractive, err := cmd.Flags().GetBool("non-interactive")
```

Comment:
What do you think about creating a struct for global objects and passing it together with config struct? Or create a wrapper struct to pass all of it as a single arg?

Reply:
I'd rather leave it as-is for now, this PR is already huge :laughing: